test/IDE/print_ast_tc_decls.swift

```diff
@@ -541,7 +541,7 @@ class d0170_TestAvailability {
 // PASS_EXPLODE_PATTERN-LABEL: {{^}}@objc class d0181_TestIBAttrs {{{$}}
 
   @IBOutlet weak var anOutlet: d0181_TestIBAttrs!
-// PASS_EXPLODE_PATTERN-NEXT: {{^}}  @IBOutlet @objc weak var anOutlet: @sil_weak d0181_TestIBAttrs!{{$}}
+// PASS_EXPLODE_PATTERN-NEXT: {{^}}  @IBOutlet @_implicitly_unwrapped_optional @objc weak var anOutlet: @sil_weak d0181_TestIBAttrs!{{$}}
```

Comment:
We don't want this attribute showing up in generated interfaces, so you should find a way to not print it here -- perhaps the UserInaccessible attribute flag does that?

Reply:
I'll take another look to see if there is a already a mechanism, and if not either add one or hack something in.

Reply:
"Implicit" does that.

Reply:
(This is pretty much the only thing "implicit" does in practice. We should probably rename it.)

Reply:
Ah, good to know.

Reply:
So it turns out this and the other test are both passing -print-implicit-attrs which is forcing these to print. That seems reasonable for a test case like this. Without that option, we aren't printing this.